spikes/cockpit-ux-spike/styles.css

```
@@ -0,0 +1,495 @@
+  font-family: var(--code-font, monospace);
+}
+.tab.on { background: var(--panel); color: var(--ink); box-shadow: inset 0 -2px 0 var(--accent); }
+.tab .x { color: var(--faint); border-radius: 3px; padding: 0 2px; }
+.tab .x:hover { background: var(--line); color: var(--ink); }
+.tab .modot { width: 6px; height: 6px; border-radius: 50%; background: var(--accent); }
+
+.editor-wrap { position: relative; flex: 1; min-height: 0; overflow: auto; background: var(--code-bg); }
+.editor-pre, .editor-ta {
+  margin: 0; padding: 14px 16px 60px 16px; font-family: var(--code-font, monospace);
+  font-size: 13px; line-height: 1.62; tab-size: 4; -moz-tab-size: 4;
+  white-space: pre; word-wrap: normal;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)